client/app/components/ActivityLogHeader/index.js

```diff
@@ -5,28 +5,43 @@ import React, { Component } from 'react'
 import { Row, Col} from 'react-bootstrap'
 //import { TSMS_TextButton } from './../../Core/Button'
 import TtnButton from 'core/Button/btn';
+import {monthArray} from '../../../constants/Index'
+import {getDate} from '../../../utils/common'
 
 
 class ActivityLogHeader extends Component{
     render(){
         let activityArray = this.props.activities;
         let totalTime = 0;
         activityArray.map((activity) => {
-            if(activity.Duration == '30 mins'){
+            if(activity.duration == '30 mins'){
                 totalTime = totalTime + 0.5;
             }
             else{
-                let duration = activity.Duration.split(' ');
+                let duration = activity.duration.split(' ');
                 let timeLog = parseInt(duration[0]);
                 totalTime = totalTime + timeLog;
             }
         })
-        //console.log('total time is =',totalTime);
+        let month = '';
+        let date = new Date(this.props.logDate);
+        let formattedDate = getDate(date);
+        let str = formattedDate.split('/');
+
+        monthArray.forEach((item) => {
+            (item.num === str[0])?month = item.month:null
+        });
+
+        let newDate = month + ' ' + str[1] + ',' + str[2];
+        let days = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
```

Comment:
Use any other way to get Day from date like moment and make a common function in utils to format date in to given format.